### DISCUSSION TITLE
How to play newly created playlists???

### DISCUSSION THREAD

**Discussion by MrSiO** [SamTV12345/PodFetch#807] [Q&A]
This feels like a stupid question, but how can I play a playlist I just created from Podfetch?

I'm able to create a playlist, add all the desired podcasts but onced created, where can I play it???  Can't find it anywhere.. it's still there under playlist, I can modify or delete it... But nowhere can I hit play.

Any info would be awesome 👍 

(Running Docker - Latest version)

**Comment by MrSiO** [comment]
So is there any way of playing back are created playlists?? Anyone?

**Reply by SamTV12345** [reply]
Normally you start the first playlist item and it should continue with the next item

**Comment by MrSiO** [comment]
Ho ok.

What will happen if one of my family member has that same first podcast in
their own playlist just like mine?

Which playlist will be played?
